lib/active_record/connection_adapters/trilogy_adapter.rb

```diff
@@ -153,6 +153,91 @@ def discard!
         self.connection = nil
       end
 
+      # ActiveRecord 7.0 support
+      if ActiveRecord.version < ::Gem::Version.new('7.1.a')
```

Comment:
There's a fair bit of logic inside this `if` statement. Would it make sense to extract the methods into its own module that we can include? That way, the 7.0 support will be nicely encapsulated.

Reply:
Your wish is fulfilled in [f500b70](https://github.com/github/activerecord-trilogy-adapter/pull/26/commits/f500b707e457bf36ac5cc5a03955656e885fb881)!

Reply:
Oh, that is nice. Thank you!